cdap-securestore-ext-cloudkms/src/main/resources/META-INF/services/co.cask.cdap.securestore.spi.SecretManager

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright © 2019 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+co.cask.cdap.securestore.spi.CloudSecretManager
```

Comment:
The implementation shouldn't go under the spi package. It should have it own specific package, such as `co.cask.cdap.securestore.gcp.cloudkms`